.github/workflows/markdown-lint.yml

```diff
@@ -0,0 +1,21 @@
+name: markdown-lint
```

Comment:
I just have one question, so this linter will be executed every time a commit is pushed right? should we maybe add a command to the Makefile so we can test it locally before pushing our changes? or should we just run it locally like we do with `golangci-lint run`

Reply:
Sure, I can add a make target for linting markdown files 👍 It won't run in every PR, only in PRs that change markdown files.

Reply:
Done 👍 I also updated the pre-push hook to lint markdown files.

Reply:
perfect! thank you